tests/test_envs.py

```diff
@@ -33,15 +44,20 @@ class TestEnvs:
 
     def test_seed(self, env: gym.Env, env_name: str):
         """Tests environment seeding."""
-        envs.test_seed(env, env_name, DETERMINISTIC_ENVS)
+        envs.test_seed(env, env_name, DETERMINISTIC_ENVS, ATARI_NO_FRAMESKIP_ENVS)
```

Comment:
Out of interest, why are these "few check" environments?

Reply:
Because it takes ~0.5 seconds to reset an atari environment, so we'd ideally like to do as few resets as possible given how many atari environments we have. For non-deterministic environments, it doesn't take that many rollouts to find a difference in how they turn out (as long as the rollouts are long, hence increase in length from 10 to 100), but for deterministic environments you'd ideally check fewer seeds with longer rollouts.

Reply:
Interesting. Half a second for a reset is a long time, I wonder why that is the case.